lib/rules/presentation-role-conflict-matches.js

```diff
@@ -0,0 +1,7 @@
+import { getImplicitRole } from '../commons/aria';
+
+function presentationRoleConflictMatches(node) {
+  return getImplicitRole(node) !== null;
```

Comment:
I think you're on the right track here, but we may need to exclude certain elements. This reminded me of #2935, where we had to account for elements that had a non-standard implicit role. The following for example probably should still be flagged by this rule:

```html
<svg role="none" aria-hidden="false">
  <title>Hello world</title>
</svg>
```

Kinda makes me wonder if I should have put that list of elements with non-standard roles somewhere that can be accessed outside the `aria-prohibited-attr` check. Maybe we need to add a toggle to `getImplicitRole()` which can return non-standard roles too? Something like:

```js
assert.isNull( getImplicitRole(svgElm) );
assert.equal( getImplicitRole(svgElm, { chromiumRoles: true ), 'SVGRoot') );
```

I think calling it `chromiumRoles` is fairly clear, and we'd have a "single source of truth" of what to return. Browsers have different role names for these elements. If we are going to do this, we'll need to make the `chromiumRoles` flag available through `getRole` as well, and we can use this in `aria-prohibited-attr`.

Reply:
I appreciate the callout here, I wasn't aware that certain elements could have a non-standard implicit role. Are those roles documented anywhere that you know of online, or was it something that mostly surfaced through testing?